src/Abstractions/TaskOrchestrationContext.cs

```diff
@@ -22,6 +22,11 @@ public abstract class TaskOrchestrationContext
     /// </summary>
     public abstract string InstanceId { get; }
 
+    /// <summary>
+    /// Gets the version of the current orchestration instance.
+    /// </summary>
+    public abstract string InstanceVersion { get; }
```

Comment:
Adding a new abstract member is a source breaking change. May need to be `virtual` and return `null` by default.

Reply:
Good suggestion, code updated, please help to review and confirm it.

And there is a similar problem in another PR but a little more complex, please have a look at: 

https://github.com/Azure/azure-functions-durable-extension/pull/2799#discussion_r1576627815